microsoft365/outlook/calendar/pkg/commands/modifyEventAttendees.go

```diff
@@ -29,7 +29,11 @@ func InviteUserToEvent(ctx context.Context, eventID, calendarID string, owner gr
 		return fmt.Errorf("failed to create client: %w", err)
 	}
 
-	if err := graph.InviteUserToEvent(ctx, c, trueEventID, trueCalendarID, owner, userEmail, message); err != nil {
+	addRequiredAttendeesList := strings.Split(addRequiredAttendees, ",")
+	addOptionalAttendeesList := strings.Split(addOptionalAttendees, ",")
+	removeAttendeesList := strings.Split(removeAttendees, ",")
```

Comment:
If any of these are not included, then the value here will be `[""]`. Is that OK? Does that cause errors?

Reply:
As long as it doesn't break anything, then we're good. I see that you already made that change, though.